### ISSUE TITLE
Revert back to upstream Dexie.Syncable

### ISSUE THREAD

**Issue by e45lee** [cs136/seashell#788]
once https://github.com/dfahlander/Dexie.js/pull/570 lands.

**Comment by yc2lee** [comment]
Looks like dfahlander/Dexie.js#570 is merged

**Comment by e45lee** [comment]
Yeah, we'll have to wait until the npm package gets updated: https://www.npmjs.com/package/dexie-syncable.
